core/src/main/java/org/mobilitydata/gtfsvalidator/notice/NoticeContainer.java

```diff
@@ -100,24 +102,25 @@ public boolean hasValidationErrors() {
   }
 
   /** Exports all validation notices as JSON. */
-  public String exportValidationNotices() {
-    return exportJson(validationNotices);
+  public String exportValidationNotices(boolean isPretty) {
+    return exportJson(validationNotices, isPretty);
   }
 
   /** Exports all system errors as JSON. */
-  public String exportSystemErrors() {
-    return exportJson(systemErrors);
+  public String exportSystemErrors(boolean isPretty) {
+    return exportJson(systemErrors, isPretty);
   }
 
   /**
    * Exports notices as JSON.
    *
    * <p>Up to {@link #MAX_EXPORTS_PER_NOTICE_TYPE} is exported per each type+severity.
    */
-  public static <T extends Notice> String exportJson(List<T> notices) {
+  public static <T extends Notice> String exportJson(List<T> notices, boolean isPretty) {
     JsonObject root = new JsonObject();
     JsonArray jsonNotices = new JsonArray();
     root.add("notices", jsonNotices);
+    Gson GSON = isPretty ? PRETTY_GSON : DEFAULT_GSON;
```

Comment:
This is not a static final, so it must be `gson` instead of `GSON`.

Reply:
True! Thanks